pubsub/redis/redis.go

```diff
@@ -92,6 +99,13 @@ func parseRedisMetadata(meta pubsub.Metadata) (metadata, error) {
 		m.password = val
 	}
 
+	if val, ok := meta.Properties[redisType]; ok && val != "" {
```

Comment:
This is a breaking change to Redis pub/sub. Can you make it so that `redisType` defaults to "node"?

Reply:
Scratch that, missed an else statement :)